### PR TITLE
admin: refactor the package structure and add swagger doc

### PR DESCRIPTION
**Pull request by fengshunli**

### CODE REVIEW

File: pkg/global/global.go [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+ */
+
+package global
+
+import (
+	"github.com/dubbogo/pixiu-admin/pkg/config"
+	"github.com/spf13/viper"
+
+	"go.uber.org/zap"
+)
```
Comment (MasterKenway) [inline pkg/global/global.go]: Is this used import format ?

**Review by MasterKenway — APPROVED** [review]

File: pkg/core/zap.go [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+		CallerKey:      "caller",
+		StacktraceKey:  global.CONFIG.Zap.StacktraceKey,
+		LineEnding:     zapcore.DefaultLineEnding,
+		EncodeLevel:    zapcore.LowercaseLevelEncoder,
+		EncodeTime:     CustomTimeEncoder,
+		EncodeDuration: zapcore.SecondsDurationEncoder,
+		EncodeCaller:   zapcore.FullCallerEncoder,
+	}
+	switch {
+	case global.CONFIG.Zap.EncodeLevel == "LowercaseLevelEncoder": // 小写编码器(默认)
```
Comment (AlexStocks) [inline pkg/core/zap.go]: 不要使用中文注释

Comment (fengshunli) [reply pkg/core/zap.go]: add fix english

File: pkg/config/gorm.go [hunk 3]
```diff
@@ -0,0 +1,17 @@
+package config
+
+type Mysql struct {
+	Path         string `mapstructure:"path" json:"path" yaml:"path"`
```
Comment (AlexStocks) [inline pkg/config/gorm.go]: host?

File: pkg/core/zap.go [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+	"github.com/dubbogo/pixiu-admin/pkg/utils"
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+)
+
+var level zapcore.Level
+
+func Zap() (logger *zap.Logger) {
+	if ok, _ := utils.PathExists(global.CONFIG.Zap.Director); !ok {
+		fmt.Printf("create %v directory\n", global.CONFIG.Zap.Director)
```
Comment (MasterKenway) [inline pkg/core/zap.go:40]: only one return value not need to be quoted

File: pkg/utils/directory.go [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+	}
+	if os.IsNotExist(err) {
+		return false, nil
+	}
+	return false, err
+}
+
+func CreateDir(dirs ...string) (err error) {
+	for _, v := range dirs {
+		exist, err := PathExists(v)
```
Comment (MasterKenway) [inline pkg/utils/directory.go:28]: The return value has the same problem

File: pkg/core/zap.go (hunk 4 above)
Comment (fengshunli) [reply pkg/core/zap.go:40]: image
zap.New what is returned is a pointer

**Review by MasterKenway — APPROVED** [review]

File: pkg/config/gorm.go (hunk 3 above)
Comment (fengshunli) [reply pkg/config/gorm.go]: fix path modify address

File: pkg/core/zap.go (hunk 4 above)
Comment (fengshunli) [reply pkg/core/zap.go:40]: variable reference

File: pkg/utils/directory.go (hunk 5 above)
Comment (fengshunli) [reply pkg/utils/directory.go:28]: Variable reference

**Review by ztelur — APPROVED** [review]

File: pkg/utils/rotatelogs_unix.go [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+package utils
+
+import (
+	"os"
+	"path"
+	"time"
+)
+
+import (
+	"github.com/dubbogo/pixiu-admin/pkg/global"
```
Comment (AlexStocks) [inline pkg/utils/rotatelogs_unix.go]: 这个 import 应该是在 第三个 import block 里面

Comment (fengshunli) [reply pkg/utils/rotatelogs_unix.go]: imports-formatter tool auto  formatted

**Review by mark4z — APPROVED** [review]

**Review by AlexStocks — APPROVED** [review]

**Pull request merged**